Problem30 - Please review

```diff
@@ -0,0 +1,25 @@
+// Time Complexity :O(n)
```

Comment:
This seems to be O(n2) solution. Can you discuss your solution in the class today please ?

Reply:
I will re-submit this PR. Will be working on it, have some confusions.